webapp/src/components/ui/calendar.tsx

```diff
@@ -54,18 +54,10 @@ function Calendar({
                 ...classNames,
             }}
             components={{
-                IconLeft: ({ className, ...props }) => (
-                    <ChevronLeft
-                        className={cn("h-4 w-4", className)}
-                        {...props}
-                    />
-                ),
-                IconRight: ({ className, ...props }) => (
-                    <ChevronRight
-                        className={cn("h-4 w-4", className)}
-                        {...props}
-                    />
-                ),
+                Chevron: ({ orientation, ...props }) => {
```

Comment:
This change comes from the react-day-picker component update, the underlying API changed in version 9)